package.json

```diff
@@ -80,9 +80,10 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "path-to-regexp": "^1.2.1",
-    "react": "^15.0.1",
-    "react-addons-test-utils": "^15.0.1",
-    "react-dom": "^15.0.1",
+    "prop-types": "^15.5.10",
+    "react": "^15.5.0",
```

Comment:
We want to keep `react`, `react-dom` and `react-addons-test-utils` at `"^15.0.1"` otherwise users who use a version of react < 15.5.0 will have to install multiple versions of react (the version they're using and version 15.5.0) which leads to large bundle sizes and bugs.